functions.php

```diff
@@ -37,19 +37,42 @@ function extractDataAndDisplay(array $data):string
     return $text;
 }
 
+function executeFormCheckAndDataInsert(array $get): string
+{
+    $message = '';
+    if (empty($_GET)) {
```

Comment:
You're using the global `$_GET` variable, but you need to be using the `$get` variable you passed in